bot_modules/irc_bot.js

```diff
@@ -105,6 +105,11 @@ command_parser = function(from, to, text, info) {
 		}
 	};
 
+	// make sure there's a string to parse! :X
+	if (typeof words[0] == 'undefined') {
+		return false;
```

Comment:
you could make this a one-liner if you want. seems ok since it's done in other parts of the code

Reply:
squashed into commit